frontend/src/utils/apiAxios.ts

```diff
@@ -37,7 +37,7 @@ export const apiAxios = () => {
   axiosInstance.interceptors.response.use(
     (response) => {
       /* On success = status code 200 */
-      return response.data;
+      return Promise.resolve(response.data);
```

Comment:
I'm not sure if this is needed but maybe it doesn't hurt. Axios' documentation on interceptors doesn't return a Promise.resolve, but just the response: https://axios-http.com/docs/interceptors

Reply:
doesn't hurt